docs/tutorial/parameter-types/date.md

```diff
@@ -0,0 +1,84 @@
+You can specify a *CLI parameter* as a Python <a href="https://docs.python.org/3/library/datetime.html" class="external-link" target="_blank">`date`</a>.
+
+Your function will receive a standard Python `date` object, and again, your editor will give you completion, etc.
+
+```Python hl_lines="2  7 8 9"
+{!../docs_src/parameter_types/date/tutorial001.py!}
+```
```

Comment:
Just a quick note here that if we do want to progress with this PR at some point, the docs should be updated with the new format, and using `Typer()` explicitely in the tutorial examples.